google-map-search.html

```diff
@@ -94,6 +94,16 @@
       },
 
       /**
+       * @deprecated
```

Comment:
this makes it look like the entire file is deprecated, when it's just the type stuff.